crates/astria-core/src/protocol/test_utils.rs

```diff
@@ -226,13 +235,19 @@ impl ConfigureSequencerBlock {
         let expanded_block_data =
             ExpandedBlockData::new_from_typed_data(&data, with_extended_commit_info).unwrap();
 
+        let rollup_data_bytes = sequence_data
```

Comment:
is this a duplicate of `rollup_data_bytes` defined previously?

Reply:
It more or less is.  The first one (which I should probably rename for clarity) is an iterator over `(&RollupId, &Bytes)` while this one is a Vec of `(RollupId, Bytes)`.

I changed this in [9bf25b6](https://github.com/astriaorg/astria/pull/2142/commits/9bf25b66bca06ac2d074ffe3ea892b981067b7fb) to create the second instance using the first one so it's clear they're essentially the same.